src/openstack_cloud/openstack_runner_manager.py

```diff
@@ -194,7 +194,7 @@ def create_runner(self, registration_token: str) -> InstanceId:
                 instance_id=instance_id,
                 image=self._server_config.image,
                 flavor=self._server_config.flavor,
-                network=self._server_config.network,
+                network=str(self._server_config.network) if self._server_config.network else None,
```

Comment:
is it no longer a str?

Reply:
It is no longer a str, it is a Url class or None

Reply:
testing locally I do not need the change.